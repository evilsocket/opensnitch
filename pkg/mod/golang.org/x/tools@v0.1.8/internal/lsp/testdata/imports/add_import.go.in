package imports //@import("package")

import (
	"fmt"
)

func _() {
	fmt.Println("")
	bytes.NewBuffer(nil)
}
