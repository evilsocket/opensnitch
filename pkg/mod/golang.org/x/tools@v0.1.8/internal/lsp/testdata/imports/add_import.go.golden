-- goimports --
package imports //@import("package")

import (
	"bytes"
	"fmt"
)

func _() {
	fmt.Println("")
	bytes.NewBuffer(nil)
}

